AmplifyTools/AmplifyCLIiOS/Sources/AmplifyCLIiOS/Core/Commands/AmplifyCommandResult.swift

```diff
@@ -0,0 +1,10 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+typealias AmplifyCommandResult = Result<[AmplifyCommandTaskResult], AmplifyCommandError>
```

Comment:
Thanks for the change -- it makes more sense to me now. Some documentation indicating that a single failed task results in an overall failed command would be useful, but the type itself is clearer and more idiomatic now.